build.sbt

```diff
@@ -1,133 +1,47 @@
-        </developer>
-        <developer>
-          <id>justinpinner</id>
-          <name>Justin Pinner</name>
-          <url>https://github.com/justinpinner</url>
-        </developer>
-      </developers>
-    ),
-  licenses := Seq("Apache V2" -> url("http://www.apache.org/licenses/LICENSE-2.0.html")),
-  publishTo := sonatypePublishToBundle.value,
-  publishConfiguration := publishConfiguration.value.withOverwrite(true)
-)
-
-lazy val checkReleaseType: ReleaseStep = ReleaseStep({ st: State =>
-  val releaseType = sys.props.get("RELEASE_TYPE").map {
-    case v if v == betaReleaseType => betaReleaseType.toUpperCase
-  }.getOrElse("PRODUCTION")
-
-  SimpleReader.readLine(s"This will be a $releaseType release. Continue? [y/N]: ") match {
-    case Some(v) if Seq("Y", "YES").contains(v.toUpperCase) => // we don't care about the value - it's a flow control mechanism
-    case _ => sys.error(s"Release aborted by user!")
-  }
-  // we haven't changed state, just pass it on if we haven't thrown an error from above
-  st
-})
-
-lazy val releaseProcessSteps: Seq[ReleaseStep] = {
-  val commonSteps = Seq(
-    checkReleaseType,
-    checkSnapshotDependencies,
-    inquireVersions,
-    runClean,
-    runTest
-  )
-
-  val prodSteps: Seq[ReleaseStep] = Seq(
-    setReleaseVersion,
-    commitReleaseVersion,
-    tagRelease,
-    publishArtifacts,
-    releaseStepCommandAndRemaining("+publishSigned"),
-    releaseStepCommand("sonatypeBundleRelease"),
-    setNextVersion,
-    commitNextVersion,
-    pushChanges
-  )
-
-  /*
-  Beta assemblies can be published to Sonatype and Maven.
-
-  To make this work, start SBT with the candidate RELEASE_TYPE variable set;
-    sbt -DRELEASE_TYPE=beta
-
-  This gets around the "problem" of sbt-sonatype assuming that a -SNAPSHOT build should not be delivered to Maven.
-
-  In this mode, the version number will be presented as e.g. 1.2.3.beta.n, but the git tagging and version-updating
-  steps are not triggered, so it's up to the developer to keep track of what was released and manipulate subsequent
-  release and next versions appropriately.
-  */
-  val betaSteps: Seq[ReleaseStep] = Seq(
-    setReleaseVersion,
-    releaseStepCommandAndRemaining("+publishSigned"),
-    releaseStepCommand("sonatypeBundleRelease"),
-    setNextVersion
-  )
-
-  commonSteps ++ (sys.props.get("RELEASE_TYPE") match {
-    case Some(v) if v == betaReleaseType => betaSteps // this enables a beta build to sonatype and Maven
-    case None => prodSteps  // our normal deploy route
-  })
-
-}
-
-val commonSettings = Seq(
+val artifactProductionSettings = Seq(
   organization := "com.gu",
-  scalaVersion := "2.13.2",
+  scalaVersion := "2.13.12",
   // scrooge 21.3.0: Builds are now only supported for Scala 2.12+
   // https://twitter.github.io/scrooge/changelog.html#id11
-  crossScalaVersions := Seq("2.12.11", scalaVersion.value),
+  crossScalaVersions := Seq("2.12.18", scalaVersion.value),
   releaseCrossBuild := true,
```

Comment:
This line needs to be moved down into the `root` project!

```suggestion
```

`releaseCrossBuild` lives with `releaseVersion` and `releaseProcess` on the root-level aggregating project, not on the artifact-producing modules!